xonsh/environ.py

```diff
@@ -29,6 +29,7 @@
     ON_CYGWIN,
     os_environ,
 )
+from xonsh.prompt.base import prompt_tokens_formatter_default
```

Comment:
This line isn't needed because of the `import xonsh.prompt.base as prompt` below